colcon_bundle/verb/_overlay_utilities.py

```diff
@@ -127,36 +107,36 @@ def recursive_tar_gz_in_path(output_path, path):
     :param path: path to recursively collect all files and include in
     tar.gz. These will be included with path as the root of the archive.
     """
+    tar_path = Path(path)
     with tarfile.open(output_path, mode='w:gz', compresslevel=5) as tar:
         logger.info(
-            'Creating tar of {path}'.format(path=path))
-        for name in os.listdir(path):
-            some_path = os.path.join(path, name)
-            tar.add(some_path, arcname=os.path.basename(some_path))
+            'Creating tar of {path}'.format(path=str(tar_path)))
+        for name in tar_path.iterdir():
+            some_path = Path(tar_path) / name
+            tar.add(str(some_path), arcname=os.path.basename(str(some_path)))
 
 
-def _generate_template(template_name, script_name, context_vars):
+def _rendering_template(template_name: str,
```

Comment:
```suggestion
def _render_template(template_name: Path,
```

My language may have been unclear. You "render" a template (like you render graphics) - in the gerund, this is "rendering" 

(and use Paths across the API - in this case it is a path relative to the assets folder but it's still a path)